### PR TITLE
[FEATURE] replace attributes by id (PUT)

### PR DESCRIPTION
**Pull request by iariasleon**
@fgalan @XavierVal

### CODE REVIEW

**Comment by XavierVal** [comment]
LGTM

File: test/acceptance/behave/README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Feature: feature name...
 
 ### Logs
 
-The log is stored in `logs` folder and is called `behave.log` see `logging.ini`.
+The log is stored in `logs` folder (if this folder does not exists it is created) and is called `behave.log` see `logging.ini`.
```
Comment (kzangeli) [inline test/acceptance/behave/README.md]: if this folder does not exist

File: test/acceptance/behave/README.md [hunk 2]
```diff
@@ -183,13 +183,13 @@ The log is stored in `logs` folder and is called `behave.log` see `logging.ini`.
...
+    * update_attribute_by_id - 654 testcases            PATCH - /v2/entities/<entity_id> plus payload
+    * replace_attributes_by_id 511 testcases            PUT - /v2/entities/<entity_id> plus payload  
+    * remove_entity (pending) 
 
   -  alarms (pending)
-
-
+  
+  
 ### Hints:
```
Comment (kzangeli) [inline test/acceptance/behave/README.md]: Do we want these added spaces?

Comment (iariasleon) [inline test/acceptance/behave/README.md]: It is something of windows to linux (replacing CRLF) , first is removed two lines and after is added two lines.

File: test/acceptance/behave/README.md (hunk 1 above)
Comment (iariasleon) [inline test/acceptance/behave/README.md]: Fixed in fd08b35

File: test/acceptance/behave/README.md (hunk 2 above)
Comment (kzangeli) [inline test/acceptance/behave/README.md]: ok.
A bit annoying but not very important

NTC

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
Find a problem here: https://github.com/telefonicaid/fiware-orion/blob/feature/replace_attributes_by_id/test/acceptance/behave/components/ngsiv2/entities/replace_attributes_by_id.feature#L2149

(I cannot comment in the file diff in the PR, as it is too large and github doesn't show the diff).

**Comment by iariasleon** [comment]
Fixed in fca466b

**Comment by fgalan** [comment]
LGTM

**Pull request merged**